Source/uart.cpp

```diff
@@ -2,12 +2,16 @@
 #include "device.h"
 #include "errorhandlers.h"
 #include "SerialBuffer.hpp"
+#ifdef STM32H743xx
+#include "stm32h7xx_hal.h"
+#else
 #include "stm32f4xx_hal.h"
+#endif
```

Comment:
should not be necessary when #including device.h, which includes hardware.h which includes device specific hal.h